types/lodash/lodash-tests.ts

```diff
@@ -5609,17 +5609,17 @@ fp.now(); // $ExpectType number
     _.chain([42]).get(0, -1); // ExpectType PrimitiveChain<number>
     _.chain({ a: { b: true } }).get("a"); // $ExpectType ObjectChain<{ b: boolean; }>
     _.chain({ a: { b: true } }).get(["a"]); // $ExpectType ObjectChain<{ b: boolean; }>
-    _.chain({ a: { b: true } }).get(["a", "b"]); // $ExpectType PrimitiveChain<false> | PrimitiveChain<true>
```

Comment:
Just noting that one of the side effects of this change is these types getting squashed so it's `boolean` instead of `true | false`. Not sure if that is problematic or not.

Reply:
I doubt it's a problem.